fxevent/event.go

```diff
@@ -78,8 +79,13 @@ type Provide struct {
 	Err error
 }
 
-// Invoke is emitted whenever a function is being invoked and/or it errored.
-type Invoke struct {
+// Invoking is emitted whenever a function is being invoked.
+type Invoking struct {
+	Function interface{}
+}
+
+// Invoked is emitted whenever a function errored.
```

Comment:
Might sound a little ambiguous as it is. How about:
```suggestion
// Invoked is emitted whenever a function being invoked errored.
```